app/services/beta/sort_stories.rb

```diff
@@ -0,0 +1,35 @@
+class Beta::SortStories
+  def initialize(sort_params)
```

Comment:
Considering you're already loading the Story itself in `Beta::StoriesController#set_project_and_story`, you could receive it here avoiding another query on database.

```ruby
def initialize(story:, params:)
  @story = story
   ...
end
```